src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -16,6 +16,14 @@ parameters:
     sylius.security.new_api_shop_regex: "^%sylius.security.new_api_shop_route%"
     sylius.security.new_api_user_account_route: "%sylius.security.new_api_shop_route%/account"
     sylius.security.new_api_user_account_regex: "^%sylius.security.new_api_user_account_route%"
+      
+sylius_api:
+    filter_eager_loading_extension:
+        restricted_operations:
+            shop_get:
+                resources:
+                    '%sylius.model.product.class%': ~
```

Comment:
can't it be simplified?

Reply:
I remember your suggestion to keep it simple by using `request_uri` from the context. I'm fine with that, but I decided to change it a little bit because I noticed one problem – `request_uri` depends on the endpoint prefix and if someone follows https://docs.sylius.com/en/1.11/customization/api/modifying_endpoints.html#how-to-modify-the-endpoints-prefixes, our workaround will stop working.